app/views/moderator_applications/_form.html.slim

```diff
@@ -0,0 +1,55 @@
+section.card
+  header.card-header
+    strong Moderator Application Form
+
+  .card-body
+    - if application.errors.any?
+      #error_explanation
+        h3 = "#{pluralize(application.errors.count, "error")} prohibited this application from being submitted:"
+        ul
+          - application.errors.messages.each do |message|
+            li = message[1][0]
```

Comment:
`message[1][0]` is an opaque accessor, which should be replaced with a more meaningful one, or at the very least commented in some way to explain what it's actually accessing. (Even I already forgot exactly what the meaning of it is, and I'm the one who wrote the code.)

Reply:
You could just drop an inline comment real quick.